buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -69,7 +69,8 @@ public final class {{class}} {
     {%- if attribute.note %}
   *
   * <p>Notes:
-    <ul> {{attribute.note | render_markdown(code="{{@code {0}}}", paragraph="<li>{0}</li>", list="{0}")}} </ul>
+        <ul> {{attribute.note | replace("> ", "") | render_markdown(code="{{@code {0}}}", paragraph="<li>{0}</li>", list="{0}")}} </ul>
```

Comment:
FYI, the `replace("> ", "")` is to remove [this](https://github.com/open-telemetry/semantic-conventions/blob/c83a10a9c33c18a769835e959200d0e24dc708fe/model/resource/k8s.yaml#L34-L38), which causes the generator to produce mangled javadoc which fails the build.